sections/1-introduction.tex

```diff
@@ -1,101 +1,76 @@
-A common platform that addresses simple tasks, provides a standard interface to data products, and encourages re-use of common functions can go a long way toward solving this problem. 
-SunPy is a project which aims to provide this solution for the field of solar physics. 
-\todo{the problem needs to be properly defined - i.e. difficult for users, no common platform, non-consistent, reproducibility etc.}
+Heliophysics as a discipline is the study of the Sun and its interactions with the solar system. 
+The Heliophysics community includes many people studying various sub-domains: the Sun, the solar wind, space weather, terrestrial and planetary magnetospheres, the heliosphere, and the Earth's ionosphere, thermosphere, and mesosphere.  
+Advancing our understanding of the fundamental processes underpinning this complex system requires interdisciplinary research across these scientific sub-disciplines.
+At the time of writing, the NASA Heliophysics System Observatory consists of 18 missions with nearly 200 instruments.
 
-The mission statement of the \sunpyproj is to facilitate and promote the use and development of a community-led, free and open-source\footnote{https://opensource.org/osd} data-analysis software based on the scientific \python\footnote{\url{https://www.python.org/}} environment. 
-To achieve that goal, the project develops and maintains a core package (\sunpypkg) and supports an ecosystem of affiliated packages (see \ref{sec:affil_packages}) that provide additional functionality. 
-The project was formally founded in March of 2014 though development of the core package began three years earlier. 
-\citet{Community:2015cy} describes version 0.5 which was released in June 2014.
+Software packages to analyze data from these instruments are generally developed independently by each instrument team. 
+This creates a diverse and therefore difficult data and software environment to navigate. 
+Scientists conducting interdisciplinary research from multiple instruments run into problems with incompatible data formats and incompatible analysis routines written with different versions of the base code. Further, it is difficult to reproduce others' scientific results without open data and version-controlled open source software.
 
-The \sunpyproj selected \python to leverage the rich ecosystem of packages available for general data analysis. 
-These include packages such as \package{Numpy} which provides efficient multi-dimensional numerical array manipulation \citep{numpy} ; \package{scipy} which provides fundamental scientific functions such as for numerical integration and optimization\citep{scipy}; \package{Matplotlib} provides publication-ready 2D plotting \citep{matplotlib}, and finally \package{Pandas} provides data structures and analysis support with support for time series \citep{pandas}.
-These packages form the foundation of the scientific \python ecosystem. 
```

Comment:
Ugh! no one is ever convinced by these metrics! This is the second SunPy paper where I try to add this and someone removes them! Though I understand that this metric is not a great one, I feel like it can be helpful to compare in a relative sense the size and scope of these projects. If folks disagree that's fine. I'll just stew in my corner. Arumph!